apps/pmp-web-e2e/src/fixtures/fixtures-example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
example of what?

Reply:
This is an example file for "general" fixtures used in e2e, currently there are github credentials used in github login command